docs/docs/getting-started/installation/installation-and-launch.md

```diff
@@ -9,9 +9,17 @@ title: Installation and launch
 
 AICA System software is distributed with Docker images and executed with Docker containers.
 
-Before proceeding, [install Docker Engine](https://docs.docker.com/engine/install/) on the host machine.
+Before proceeding, you will need to install Docker Engine on the host machine. Depending on your platform, the
+installation process may vary or require additional steps. Below, you will find instructions per operating system:
 
-:::info
+<details>
+<summary>Linux</summary>
+
+AICA suggests Ubuntu Linux as the main platform, but other distributions where Docker can be installed work equally as 
+well. You may find distribution-specific instructions to install Docker Engine [here](https://docs.docker.com/engine/install/).
+Note that you can and it is **recommended** that you skip the installation of Docker Desktop for Linux.
+
+:::note
 
 For Ubuntu users, make sure to follow
 the [post installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to create the `docker` group
```

Comment:
We could change this link to https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user